.circleci/config.yml

```diff
@@ -273,7 +273,15 @@ build_and_deploy: &build_and_deploy
             if grep -Fxq "$PANTHEON_ENV" ./env_list.txt; then
               echo "Multidev already exists, no need to reset index tracker"
             else
-              echo "Reseting index tracker because index is empty on new multidev"
+              echo "Resetting Solr server on new multidev"
```

Comment:
Fixed issue with new multidevs not indexing due to `Solr error: cannot change field "tm_X3b_en_rendered_item" from index options=DOCS_AND_FREQS_AND_POSITIONS to inconsistent index options=DOCS_AND_FREQS_AND_POSITIONS_AND_OFFSETS` error